### PR TITLE
Exode pr Arc7_3G

### PR DESCRIPTION
**Pull request by Xode672**

### CODE REVIEW

File: script/Arcueid/Day 7/07_00_ARC07_3G.txt
Comment (rschlaikjer) [inline script/Arcueid/Day 7/07_00_ARC07_3G.txt:4]: Hmm, feels off to me. Would revert.

File: script/Arcueid/Day 7/07_00_ARC07_3G.txt
Comment (rschlaikjer) [inline script/Arcueid/Day 7/07_00_ARC07_3G.txt:59]: Does drop the 今度 slightly (the `now`), but more importantly the placement of the comma is now off:
> This girl's just standing there, watching me an outsized grin on her face.

If keeping the change you'd want 
> This girl's just standing there watching me, an outsized grin on her face.

File: script/Arcueid/Day 7/07_00_ARC07_3G.txt
Comment (rschlaikjer) [inline script/Arcueid/Day 7/07_00_ARC07_3G.txt:107]: Location singular makes sense here I think, if the dead patrol routes overlapped all the kills would be around the one area.

File: script/Arcueid/Day 7/07_00_ARC07_3G.txt
Comment (rschlaikjer) [inline script/Arcueid/Day 7/07_00_ARC07_3G.txt:331]: I'd revert - the flow of this sentence isn't really natural, it cuts from an active voice ("I take off") to more passive "with my glasses removed" with only a comma separating the two. The JP flow is also more serial / active (taking off his glasses, rather than jumping to the state of them being removed).

File: script/Arcueid/Day 7/07_00_ARC07_3G.txt
Comment (rschlaikjer) [inline script/Arcueid/Day 7/07_00_ARC07_3G.txt:439]: The JP is a bit more vague here on presence, since it doesn't really attach a subject (literally approx "Presence is a thing which remains in a place."), which is why it was left as 'someone'.

File: script/Arcueid/Day 7/07_00_ARC07_3G.txt
Comment (rschlaikjer) [inline script/Arcueid/Day 7/07_00_ARC07_3G.txt:484]: I don't know the linguistic name for it, but "see if we can't X" and "see if we can X" can both mean the same thing in English :upside_down_face: 
I guess if people are liable to think it's wrong then better to be safe.

File: script/Arcueid/Day 7/07_00_ARC07_3G.txt
Comment (rschlaikjer) [inline script/Arcueid/Day 7/07_00_ARC07_3G.txt:587]: Try and avoid adding these trailing spaces, re-exporting the script using luna_cli is a way to ensure everything is formatted correctly.

File: script/Arcueid/Day 7/07_00_ARC07_3G.txt
Comment (rschlaikjer) [inline script/Arcueid/Day 7/07_00_ARC07_3G.txt:827]: Yeah it's intentionally vague here to allow the subsequent dialog. 

Not sure I love this use of without a doubt, it doesn't really feel right with the pause structure it's used in here.

File: script/Arcueid/Day 7/07_00_ARC07_3G.txt
Comment (rschlaikjer) [inline script/Arcueid/Day 7/07_00_ARC07_3G.txt:860]: Kinder prefer the active form here

File: script/Arcueid/Day 7/07_00_ARC07_3G.txt
Comment (rschlaikjer) [inline script/Arcueid/Day 7/07_00_ARC07_3G.txt:889]: I think the 'potential' form (`are`) makes more sense than `were` here since I don't think we're talking about a specific instance of a person in the past, but making a general statement that continues to be true.

**Review by rschlaikjer — CHANGES_REQUESTED** [review]

File: script/Arcueid/Day 7/07_00_ARC07_3G.txt
Comment (Xode672) [reply script/Arcueid/Day 7/07_00_ARC07_3G.txt:107]: Hmm I thought that making it locations was to more emphasize that there would be multiple locations where the killings may gather making the problem seem more wide scale. Do we have an idea on how big Roa's operation was?

File: script/Arcueid/Day 7/07_00_ARC07_3G.txt
Comment (Xode672) [reply script/Arcueid/Day 7/07_00_ARC07_3G.txt:484]: Changed it because it took me a while to get it lol, if it's just me then I think reverting is fine

File: script/Arcueid/Day 7/07_00_ARC07_3G.txt
Comment (Avuxo) [inline script/Arcueid/Day 7/07_00_ARC07_3G.txt:156]: maybe "the bare minimum for subsistence"

File: script/Arcueid/Day 7/07_00_ARC07_3G.txt
Comment (Avuxo) [inline script/Arcueid/Day 7/07_00_ARC07_3G.txt:331]: remove trailing space?

File: script/Arcueid/Day 7/07_00_ARC07_3G.txt
Comment (Avuxo) [reply script/Arcueid/Day 7/07_00_ARC07_3G.txt:587]: this was marked resolved but the trailing space is still here.

**Review by Avuxo — APPROVED** [review]
pending changes, lgtm

**Review by rschlaikjer — APPROVED** [review]
LGTM pending Ben's comments

**Pull request merged**
